p2p_wallet/Coordinators/TabBar/HomeCoordinator.swift

```diff
@@ -118,8 +118,8 @@ final class HomeCoordinator: Coordinator<Void> {
             }
         case .receive(let publicKey):
             let coordinator = ReceiveCoordinator(navigationController: navigationController, pubKey: publicKey)
-            analyticsManager.log(event: AmplitudeEvent.mainScreenReceiveOpen)
-            analyticsManager.log(event: AmplitudeEvent.receiveViewed(fromPage: "main_screen"))
+            analyticsManager.log(event: .mainScreenReceiveOpen)
+            analyticsManager.log(event: .receiveViewed(fromPage: "main_screen"))
```

Comment:
it is interesting that somewhere we use "Main_screen" and here it is "main_screen". I wonder if we can extract this into enum or smth like that

Reply:
Yes. Let's create another task and discuss it with data analyst team later to make it more standard